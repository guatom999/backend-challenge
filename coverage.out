mode: set
github.com/guatom999/backend-challenge/main.go:11.13,21.2 4 0
