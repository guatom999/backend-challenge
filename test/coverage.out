mode: set
github.com/guatom999/backend-challenge/test/setup.go:5.37,8.2 2 1
